grades/README.md

```diff
@@ -185,3 +185,39 @@ the proctored exam being released. So you will need to run the management comman
 When a learner's fifth and final course certificate is created, it triggers the creation of a program certificate. 
 Course certificates are generated every hour (on the hour). If this is too long to wait, use the 
 `generate_program_certificates` management command. Each learner can only earn one Program certificate. 
+
+
+##Sync edx current cached grade and FinalGrade
+It happens that after the grades were frozen, account relinking can occur 
+which results in grades being out of sync, since `FinalGrade` doesn't get updated
+after grades had been frozen.
```

Comment:
Slight wording suggestion

```suggestion
Occasionally, a learner's grade may need to be updated manually. One way this 
could happen is if a learner's MITx Online account was linked to their Micromasters 
account after some grades were frozen. This would result in grades being out of sync, 
since `FinalGrade` doesn't get updated after grades had been frozen.
```